src/app/search/directives/search-story.component.ts

```diff
@@ -1,96 +1,59 @@
-import { Component, Input, OnInit } from '@angular/core';
-
-import { HalDoc } from '../../core';
-import { StoryModel } from '../../shared';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'publish-search-story',
   styleUrls: ['search-story.component.css'],
   template: `
-    <div>
-      <section class="story-image">
-        <publish-image [imageDoc]="story.doc"></publish-image>
-        <p *ngIf="statusClass" [class]="statusClass">{{statusText}}</p>
-      </section>
-      <section class="story-detail">
-        <h2 class="story-title"><a [routerLink]="editStoryLink">{{storyTitle}}</a></h2>
-        
-        <h3 *ngIf="seriesLink" class="series-title"><a [routerLink]="seriesLink">{{seriesTitle}}</a></h3>
-        <h3 *ngIf="!seriesLink" class="series-title">{{seriesTitle}}</h3>
-        
-        <p class="meta">
-          <span class="duration">{{storyDuration | duration}}</span>
-          <span *ngIf="storyAudioTotal" class="audio-total"><i class="icon-up-dir"></i>{{storyAudioTotal}}</span>
-          <span class="modified">{{storyUpdated | date:"MM/dd/yy"}}</span>
-        </p>
-      </section>
-      <section class="story-tags">
-        <span *ngFor="let tag of storyTags">{{tag}}</span>          
-      </section>
-      <section class="story-description three-lines" [ngClass]="{'two-lines': storyTags.length > 0 }">
-        {{storyDescription}}
-      </section>
+    <div class="form-group">
+      <!-- TODO: delay search until finish typing -->
+      <input (keyup)="searchByText()" [(ngModel)]="searchText" placeholder="search by title or description"/>
+    </div>
+    
+    <div class="form-group">
+      <p class="left">
+        <label [attr.for]="searchSeries">Filter by Series</label>
+        <select id="searchSeries" [(ngModel)]="searchSeriesId" (ngModelChange)="searchBySeries()">
+          <option *ngFor="let seriesId of allSeriesIds" [value]="seriesId">{{allSeries[seriesId]?.title || 'No Series'}}</option>
+        </select>
+        <!-- TODO: there should be a way to clear this -->
```

Comment:
...but that feels so 2005 :smiley: How about opening a ticket for adding a select2 implementation in place of selects?